### PR TITLE
[ISSUE #6358]Add chinese docs about persistent-unique-broker-id

### LINKED ISSUE #6358: Add chinese docs about persistent-unique-broker-id
Add chinese docs about persistent-unique-broker-id

The issue tracker is used for bug reporting purposes **ONLY** whereas feature request needs to follow the [RIP process](https://github.com/apache/rocketmq/wiki/RocketMQ-Improvement-Proposal). To avoid unnecessary duplication, please check whether there is a previous issue before filing a new one.

It is recommended to start a discussion thread in the [mailing lists](http://rocketmq.apache.org/about/contact/) or [github discussions](https://github.com/apache/rocketmq/discussions) in cases of discussing your deployment plan, API clarification, and other non-bug-reporting issues.
We welcome any friendly suggestions, bug fixes, collaboration, and other improvements.

Please ensure that your bug report is clear and self-contained. Otherwise, it would take additional rounds of communication, thus more time, to understand the problem itself.

Generally, fixing an issue goes through the following steps:
1. Understand the issue reported;
1. Reproduce the unexpected behavior locally;
1. Perform root cause analysis to identify the underlying problem;
1. Create test cases to cover the identified problem;
1. Work out a solution to rectify the behavior and make the newly created test cases pass;
1. Make a pull request and go through peer review;

As a result, it would be very helpful yet challenging if you could provide an isolated project reproducing your reported issue. Anyway, please ensure your issue report is informative enough for the community to pick up. At a minimum, include the following hints:

**BUG REPORT**

1. Please describe the issue you observed:

- What did you do (The steps to reproduce)?

- What is expected to see?

- What did you see instead?

2. Please tell us about your environment:

3. Other information (e.g. detailed explanation, logs, related issues, suggestions on how to fix, etc):

**FEATURE REQUEST**

1. Please describe the feature you are requesting.

2. Provide any additional detail on your proposed use case for this feature.

3. Indicate the importance of this issue to you (blocker, must-have, should-have, nice-to-have). Are you currently using any workarounds to address this issue?

4. If there are some sub-tasks involved, use -[] for each sub-task and create a corresponding issue to map to the sub-task:

- [sub-task1-issue-number](example_sub_issue1_link_here): sub-task1 description here, 
- [sub-task2-issue-number](example_sub_issue2_link_here): sub-task2 description here,
- ...

### PR DESCRIPTION
**Pull request by TheR1sing3un**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

1. add chinese docs about persistent-unique-broker-id
closes #6358 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/cn/controller/persistent_unique_broker_id.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+```
+> 启动一个旧版controller
+
+```bash
+nohup sh bin/mqcontroller -c ./conf/controller/controller-standalone.conf &
+```
+> 查看controller是否被正确启动
+
+```bash
+sh bin/mqadmin getControllerMetaData -a localhost:9878
```
Comment (RongtongJin) [inline docs/cn/controller/persistent_unique_broker_id.md:104]: 测试相关内容去掉吧

File: docs/cn/controller/persistent_unique_broker_id.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+![](../image/controller/persistent_unique_broker_id/register_process.png)
+
+#### 1. GetNextBrokerId Request
+这时候发起一个`GetNextBrokerId`的请求到Controller，为了拿到当前的下一个待分配的`BrokerId`(从1开始分配)。
+#### 1.1 ReadFromDLedger
+此时Controller接收到请求，然后走DLedger去获取到状态机的`NextBrokerId`数据。
+#### 2. GetNextBrokerId Response
+Controller将`NextBrokerId`返回给Broker。
+#### 2.1, 2.2 CreateTempMetaFile
+Broker拿到`NextBrokerId`之后，创建一个临时文件`.broker.meta.temp`，里面记录了`NextBrokerId`(也就是期望应用的`BrokerId`)，以及自己生成一个`RegisterCode`(用于之后的身份校验)也持久化到临时文件中。
```
Comment (mxsm) [inline docs/cn/controller/persistent_unique_broker_id.md:21]: 需要删除2.2

File: docs/cn/image/controller/persistent_unique_broker_id/normal_restart.png
Comment (mxsm) [inline docs/cn/image/controller/persistent_unique_broker_id/normal_restart.png:1]: 这个图能否重新处理一下

File: docs/cn/image/controller/persistent_unique_broker_id/test_1.png
Comment (mxsm) [inline docs/cn/image/controller/persistent_unique_broker_id/test_1.png:1]: 这个图是否可以重新截图整理

File: docs/cn/image/controller/persistent_unique_broker_id/test_10.png
Comment (mxsm) [inline docs/cn/image/controller/persistent_unique_broker_id/test_10.png:1]: 这个图是否可以重新截图整理

File: docs/cn/image/controller/persistent_unique_broker_id/test_12.png
Comment (mxsm) [inline docs/cn/image/controller/persistent_unique_broker_id/test_12.png:1]: 这个图是否可以重新截图整理

File: docs/cn/image/controller/persistent_unique_broker_id/test_13.png
Comment (mxsm) [inline docs/cn/image/controller/persistent_unique_broker_id/test_13.png:1]: 这个图是否可以重新截图整理

File: docs/cn/image/controller/persistent_unique_broker_id/test_14.png
Comment (mxsm) [inline docs/cn/image/controller/persistent_unique_broker_id/test_14.png:1]: 这个图是否可以重新截图整理

**Review by mxsm — CHANGES_REQUESTED** [review]
@TheR1sing3un  文档的结构标题排版需要在处理一下，以及后面的这个图片需要重新整理一下。看起来更加正式一点

**Review by RongtongJin — APPROVED** [review]

**Review by mxsm — APPROVED** [review]
LGTM~

**Pull request merged**